docs/01_nodeos/07_concepts/15_private_chain_access/index.md

```diff
@@ -0,0 +1,79 @@
+
+The C++ security_group API exposes the following functions to be used within smart contracts. You can add or remove participants, check existence within the *security group*, or get a list of the current participants:
+
+* [`add_security_group_participants()`](https://developers.eos.io/manuals/eosio.cdt/latest/namespaceeosio_1_1internal__use__do__not__use#function-add_security_group_participants)
+* [`remove_security_group_participants()`](https://developers.eos.io/manuals/eosio.cdt/latest/namespaceeosio_1_1internal__use__do__not__use#function-remove_security_group_participants)
+* [`in_active_security_group()`](https://developers.eos.io/manuals/eosio.cdt/latest/namespaceeosio_1_1internal__use__do__not__use#function-in_active_security_group)
+* [`get_active_security_group()`](https://developers.eos.io/manuals/eosio.cdt/latest/namespaceeosio_1_1internal__use__do__not__use#function-get_active_security_group)
+
+Note that adding or removing participants within the *security group* requires a privileged account. Also, note that these two actions are "proposed" for approval via EOSIO consensus. Finally, checking for existence of participants in the *security group* is all or nothing: if there is at least one specified participant that is not in the *security group*, the check returns false.
+
+For more information visit the [C++ security_group API](https://developers.eos.io/manuals/eosio.cdt/latest/security__group_8hpp) reference.
+
+## Examples
+
+The examples below showcase how to enable the SECURITY_GROUP protocol feature as well as adding and removing participants from the *security group*:
+* [Private Access Control Tutorial](05_tutorial.md)
+* [How to add participants to security group](10_how-to-add-participants-to-security-group.md)
+* [How to remove participants to security group](15_how-to-remove-participants-to-security-group.md)
+
+## Use Cases
+
+Below are some of the potential use cases for *Private Access Control*, along with applicable business privacy requirements:
+
+* Consortium Solutions:
+  - Control on-chain user access and control via smart contracts.
+  - Allow storage and transmission of encrypted data among consortium members.
+  - Maintain a synced registry of approved vendors with all participants of the chain, with transparency on when a record is added or edited.
+  - Transaction recording and tracking using hash references of private chain data to a parent chain of consortium members.
+
+* Supply Chain Solutions:
+  - Private messaging between Clients to exchange confidential messages between trading partners by leveraging EPCIS technology and standards.
+  - Blockchain as a shared, immutable ledger to register the proof of the authenticity of transactions and execute smart contracts. The blockchain will enforce business rules, such as only one company can have legal ownership of a serialized unit at a given time.
+  - zk-SNARKs or another method of enhancing privacy to ensure no business data is revealed.
```

Comment:
I think we can spell out the acronym and add the acronym in brackets:

Zero-Knowledge Succinct Non-Interactive Argument of Knowledge (zk-SNARKs)

That's a mouthful! :-) 

Or just hyperlink the acronym to an authoritative external page.